setup.py

```diff
@@ -6,6 +6,9 @@
 with open("requirements.txt") as f:
     INSTALL_REQUIRES = f.read().strip().split("\n")
 
+EXTRAS_REQUIRE = {"viz": ["geoviews >= 1.9.0", "cartopy >= 0.18.0"]}
```

Comment:
Thanks for helping me understand @weiji14, I figured out the reason why it kept failing was due to incorrect GDAL configuration in my OS (gdal is not in PATH), but it was solved now. The only issue is that`geoviews` requires `scipy` as a dependency,  which will be solved by adding `scipy` in `EXTRAS_REQUIRE`: 
`EXTRAS_REQUIRE = {"viz": ["geoviews >= 1.9.0", "cartopy >= 0.18.0", "scipy"]}`. After this I think we are good to  merge and move onto https://github.com/icesat2py/icepyx/pull/176.

Reply:
Ok, done in 5be44370d99b99e638b2a428b0b1c68bb40c7e6e